lighthouse-core/gather/driver.js

```diff
@@ -464,6 +467,25 @@ class Driver {
     }
   }
 
+  /**
+   * Evaluate an expression (optionally defined in a structured manner, see `createEvalCode`
```

Comment:
> see `createEvalCode`

this is the interface exposed to gathering. Are there any docs on `createEvalCode` that could move here?

Reply:
No. But we could copy the contents of createEvalCode here?

I'm surprised `@see` doesn't do anything useful in vscode.

Reply:
ref https://github.com/microsoft/TypeScript/issues/35524